examples/rpc/README.md

```diff
@@ -0,0 +1,33 @@
+# RPC Demo
+
+A very simple example to show peer to peer communication with Godot's high level 
+
+multiplayer API calls.
+
+
+## Setup
+
+1. Build the library:
+
+    `cargo build`
+
+2. Import the Godot project in the Godot editor.
+
+3. Open the `Server.tscn` scene and start the current scene with `F6`.
+
+4. Start a second instance of the Godot editor.
+
+5. In the `Remote Port` in the Debug settings of the editor (e.g. 6012).

```

Comment:
Missing verb and confusing grammar. Suggestion: Set a separate "Remote Port" for the second editor (e.g. 6012).

Reply:
Thanks, not sure what I did here. I have made the changes. Please check again when you have time.